cpp/include/kvikio/file_handle.hpp

```diff
@@ -66,7 +70,13 @@ inline int open_fd_parse_flags(const std::string& flags, bool o_direct)
     default: throw std::invalid_argument("Unknown file open flag");
   }
   file_flags |= O_CLOEXEC;
-  if (o_direct) { file_flags |= O_DIRECT; }
+  if (o_direct) {
+#if defined(O_DIRECTT)
```

Comment:
Looks like a small typo snuck in here. Fixing in PR: https://github.com/rapidsai/kvikio/pull/416

```suggestion
#if defined(O_DIRECT)
```